### PR TITLE
[feature] Setup API for configers

### PR DESCRIPTION
**Pull request by za4emnik**
https://garage.atlassian.net/browse/PEO-65

### CODE REVIEW

File: exe/inquisition [hunk 1]
```diff
@@ -4,6 +4,10 @@ require 'inquisition'
 
 begin
   Inquisition::CLI.start(ARGV)
-rescue Inquisition::Errors::BaseConfigAbsenseError => e
+rescue Inquisition::Errors::BaseConfigAbsenseError,
```
Comment (dzemlianoi-double) [inline exe/inquisition]: separate constant

File: lib/inquisition/core/checker.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+      private
+
+      def run_cli(command)
+        begin
+          stdout, stderr, status = Open3.capture3(command)
+        rescue Errno::ENOENT
+          Outputer.additional_software_not_found(command)
+        end
+        { stdout: stdout || '', stderr: stderr, status: status }
```
Comment (dzemlianoi-double) [inline lib/inquisition/core/checker.rb]: use rspec-command

File: spec/auditors/backend/fasterer/installer_spec.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe Inquisition::Auditors::Backend::Fasterer::Installer do
+  # describe '#config_name' do
```
Comment (dzemlianoi-double) [inline spec/auditors/backend/fasterer/installer_spec.rb]: check

File: spec/core/configers/copy_file_configer_spec.rb [hunk 4]
```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
```
Comment (dzemlianoi-double) [inline spec/core/configers/copy_file_configer_spec.rb:1]: fill

File: spec/core/configers/update_file_configer_spec.rb [hunk 5]
```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
```
Comment (dzemlianoi-double) [inline spec/core/configers/update_file_configer_spec.rb:1]: fill

**Review by dzemlianoi-double — APPROVED** [review]

**Comment by dzemlianoi-double** [comment]
fix and merge

**Comment by za4emnik** [comment]
merge without specs for each linter

**Pull request merged**